uvm/servlets/admin/config/network/view/Interfaces.js

```diff
@@ -260,7 +261,13 @@ Ext.define('Ung.config.network.view.Interfaces', {
                 dataIndex: 'address',
                 width: Renderer.ipWidth,
                 flex: 1
-            }],
+            },{
+                header: 'Vendor'.t(),
```

Comment:
Let's clarify as "MAC Vendor".  Or maybe do a grouping of MAC address/vendor like we do under Devices.

Reply:
Done